msm/src/proof.rs

```diff
@@ -45,6 +43,19 @@ impl<const N: usize, G: KimchiCurve> ProofInputs<N, G, LookupTableIDs> {
     }
 }
 
+impl<const N: usize, G: KimchiCurve, ID: LookupTableID> Default for ProofInputs<N, G, ID> {
+    fn default() -> Self {
```

Comment:
I created this because in main we used a default to create the proof inputs. But probably we won't need it in the final version and will be able to delete it eventually.